src/interfacemain.cpp

```diff
@@ -81,7 +81,7 @@ int main( int argc, char *argv[] )
         if( !line.isNull() ) {
 	    if (line.at( 0 ) == 'N' ) {
 		repoAlias = line.split( " " ).at( 1 );
```

Comment:
I had a look and saw that is parsing what is written with:
outData << "N " << repo->name() << "\n";
Since repo->name() might contain spaces, it's not right to use line.split(" ").at( 1 ); as that might get just part of the name. I think it's better to use line.mid(2); to just ignore the first two characters.